pkg/csi_driver/controller.go

```diff
@@ -53,8 +53,10 @@ const (
 	enterpriseTierMaxSize = 10 * util.Tb
 	highScaleTierMinSize  = 10 * util.Tb
 	highScaleTierMaxSize  = 100 * util.Tb
-	zonalTierMinSize      = 1 * util.Tb
-	zonalTierMaxSize      = 100 * util.Tb
+	zonalLowTierMinSize   = 1 * util.Tb
```

Comment:
I would recommend to use the same terminology of 'small' and 'large' instead of 'low' and 'high' to make it easy to read and understand.

Reply:
Good suggestion. Done.